src/functions/auth.ts

```diff
@@ -0,0 +1,19 @@
+import firebase from "firebase/app";
```

Comment:
This file fits into the intent of our existing `src/api/` instead of `src/functions/`